modules/govuk/lib/puppet/parser/functions/data_sync_times.rb

```diff
@@ -0,0 +1,38 @@
+module Puppet::Parser::Functions
+  newfunction(:data_sync_times, :type => :rvalue, :doc => <<-EOS
+Return information about the data sync times, depending on the argument passed
+    EOS
+  ) do |args|
+    start_hour = 22
+    start_minute = 0
+    finish_hour = 8
+    finish_minute = 0
+
+    if args.size != 1
+      raise(ArgumentError, "data_sync_times: Wrong number of arguments " +
+      "(given #{args.size})")
```

Comment:
```suggestion
      raise ArgumentError, 
            "data_sync_times: Wrong number of arguments " + "(given #{args.size})"
```

It seems a bit unusual to have braces on a raise in Ruby since it's treated more as a keyword than a method. I've no idea if puppet does anything special here.